aries-site/src/layouts/content/Example/Example.js

```diff
@@ -106,14 +107,23 @@ export const Example = ({
   else ExampleWrapper = Box;
 
   let viewPort;
+  let scaledTheme;
+  // 0.5 selected to reduce breakpoints and spacing by half. This can be
+  // enhanced in the future by changing to a state variable which can be
+  // manipulated by a control presented to the user.
+
   if (screen === screens.mobile) viewPort = 'small';
   else if (!screenContainer && !showResponsiveControls) viewPort = size;
   else if (screenContainer) {
     if (fullscreen) viewPort = size;
     else if (!fullscreen) {
       const containerWidth =
         inlineRef.current && inlineRef.current.getBoundingClientRect().width;
-      const { breakpoints } = theme.global;
+      scaledTheme = screenContainer.scale
+        ? scaled(theme, screenContainer.scale)
+        : theme;
```

Comment:
We fallback to `theme` here but also fallback to `theme` on line 161. Do we need both? Or could this just be:
```
scaledTheme = screenContainer.scale && scaled(theme, screenContainer.scale)
```

Reply:
mmmm... good question.  I think if we did that, we'd need to initialize scaledTheme on L110 as `let scaledTheme = theme;`, otherwise we'll get an undefined `viewport`. I'll go back and revisit.

Reply:
Got it, not critical. just a thought.